tests/unit/test_backends.py

```diff
@@ -12,179 +12,142 @@
-        self.assertTrue(isinstance(response, class_))
-
-    def testSearchVariantSets(self):
-        request = protocol.SearchVariantSetsRequest()
-        request.datasetId = self.getDataset().getId()
-        responseStr = self._backend.runSearchVariantSets(
-            request.toJsonString())
-        response = protocol.SearchVariantSetsResponse.fromJsonString(
-            responseStr)
-        self.assertTrue(
-            isinstance(response, protocol.SearchVariantSetsResponse))
-
-    def testSearchVariants(self):
-        variantSetIds = [
-            variantSet.id for variantSet in self.getVariantSets(pageSize=1)]
-        request = protocol.SearchVariantsRequest()
-        request.variantSetId = variantSetIds[0]
-        responseStr = self._backend.runSearchVariants(request.toJsonString())
-        response = protocol.SearchVariantsResponse.fromJsonString(
-            responseStr)
-        self.assertTrue(
-            isinstance(response, protocol.SearchVariantsResponse))
-
-    def testSearchCallSets(self):
-        variantSetIds = [
-            variantSet.id for variantSet in self.getVariantSets(pageSize=1)]
-        request = protocol.SearchCallSetsRequest()
-        request.variantSetId = variantSetIds[0]
-        responseStr = self._backend.runSearchCallSets(request.toJsonString())
-        response = protocol.SearchCallSetsResponse.fromJsonString(
-            responseStr)
-        self.assertTrue(
-            isinstance(response, protocol.SearchCallSetsResponse))
-
-    def testVariantSetPagination(self):
-        results = []
-        for pageSize in range(1, 100):
-            variantSetIds = [
-                variantSet.id for variantSet in self.getVariantSets(
-                    pageSize=pageSize)]
-            results.append(variantSetIds)
-        for result in results[1:]:
-            self.assertEqual(result, results[0])
-
-    def runListReferenceBases(self, id_):
-        requestArgs = {"start": 3, "end": 5, "pageToken": "0"}
-        responseStr = self._backend.runListReferenceBases(id_, requestArgs)
-        response = protocol.ListReferenceBasesResponse.fromJsonString(
-            responseStr)
-        self.assertTrue(
-            isinstance(response, protocol.ListReferenceBasesResponse))
-
-    def testRunListReferenceBases(self):
-        referenceSet = self._backend.getReferenceSets()[0]
-        reference = referenceSet.getReferences()[0]
-        self.runListReferenceBases(reference.getId())
-
-    def testDatasetNotFound(self):
-        request = protocol.SearchVariantSetsRequest()
-        datasetId = 'doesNotExist'
-        request.datasetId = datasetId
-        with self.assertRaises(exceptions.DatasetNotFoundException):
-            self._backend.getDataset(request.datasetId)
-
-
-class TestFileSystemBackend(TestAbstractBackend):
+            self._backend.getReferenceSetByName(referenceSet.getLocalId()),
+            referenceSet
+        )
```

Comment:
We normally put the closing bracket on the same line when doing hanging indents like this. It saves a bit of vertical space. There's a few places below where it might be nice to do this (but optional, of course, since it passes pep8).